gradoop-data-integration/src/main/java/org/gradoop/dataintegration/importer/rdbmsimporter/tuples/RowHeaderTuple.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradoop.dataintegration.importer.rdbmsimporter.tuples;
+
+import org.apache.flink.api.java.tuple.Tuple3;
+
+/**
+ * Represents a tuple of a rowheader
+ */
+public class RowHeaderTuple extends Tuple3<String, String, Integer> {
+
+  /**
+   * serial version uid
+   */
+  private static final long serialVersionUID = 1L;
+
+  /**
+   * Name of attribute
+   */
+  private String name;
+
+  /**
+   * Data type of attribute
+   */
+  private String attType;
+
+  /**
+   * Position of attribute in row
+   */
+  private int pos;
+
+  /**
+   * Empty Constructor
+   */
+  public RowHeaderTuple() {
```

Comment:
see above about tuple creation and usage of super constructor.

Reply:
Done